Main.java

```diff
@@ -1,21 +1,90 @@
 class Main {
   public static void main(String[] args) {
-   // kindly use appropriate data types for your declaration
-   // declare variables days the debtor defaulted, amount the debtor is to pay per day, amount the debtor is to pay, amount the debtor paid and assign values to them
-   
-   //Declare the variable of the total amount the debtor is to pay, the days the debtor paid for, amount the debtor is left to pay, days the debtor has not payed for
-
-   // calculate and print total amount the debtor is to pay
-
-   // calculate and print the days the debtor paid for
-  
-  // calculate and print the amount whose day was not captured because the amount was incomplete
-
-   // calculate and print amount the debtor is left to pay
-
-   // calculate and print days the debtor has not payed for
-
-   //kindly remove the statement below when you are done with the assignment
-    System.out.println("Hello world!");
+    float 	amountToPayDaily = 100.00f;					  //amount debtor opted to pay back per day
+		 float 	DebtOwed = 500.00f;							      //total amount owed by debtor
```

Comment:
This is meant to be calculated from the amount the debtor is meant to pay daily and the days defaulted

Reply:
i don't think i understand what you mean sir.